src/app/to-do/to-do.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-to-do',
+  templateUrl: './to-do.component.html',
+  styleUrls: ['./to-do.component.css']
+})
+export class ToDoComponent {
+  public items: string[] = [];
+
+  public newTask: any;
```

Comment:
Avoid the use of `any`. When you see `any` you should always look for an actual type to associate with it. Even if it can be multple types you can specify that too. When we use any in TypeScript we lose all value of using TypeScript because it cannot provide any help or linting for us. In this case it's probably meant to be a `string`.

Reply:
This has been corrected in most recent commits.